processes/dags/pubmed_migration.py

```diff
@@ -111,59 +105,247 @@ def make_tarfile(output_filepath: str, source_dir: str) -> None:
         tar.add(source_dir, arcname=os.path.basename(source_dir))
 
 
+def delete_temp() -> None:
+    """Deletes temporary directory that processes files
+    """
+    root_dir = '/usr/local/airflow'
+    temp_dir = root_dir + '/' + 'temp'
+    delete_dir(temp_dir)
+
+
+def make_jsonfile(obj: dict, output_filepath: str) -> None:
+    """Make a JSON file from the casereport dictionary
+
+    Args:
+        obj (dict): casereport XML parsed as a dictionary
+        output_filepath (str): path of output file.
+    """
+    with open(output_filepath, 'w') as json_file:
+        json.dump(obj, json_file)
+
+
+def make_case_report_json(xml_path: str, json_path: str) -> None:
+    """Makes a JSON file from pubmed XML files
+
+    Args:
+        xml_path (str): path to input XML file
+        json_path (str): path to destination JSON file
+    """
+    pubmed_xml = pp.parse_pubmed_xml(xml_path)
+    pubmed_paragraph = pp.parse_pubmed_paragraph(xml_path)
+    pubmed_references = pp.parse_pubmed_references(xml_path)
+
+    text = " ".join([p["text"] for p in pubmed_paragraph])
+    case_report = {
+        "pmID": pubmed_xml["pmid"],
+        "messages": [],
+        "source_files": [],
+        "modifications": [],
+        "normalizations": [],
+        # ctime            : 1351154734.5055847,
+        "text": text,
+        "entities": [],
+        "attributes": [],
+        # date : { type: Date, default: Date.now }
+        "relations": [],
+        "triggers": [],
+        "events": [],
+        "equivs": [],
+        "comments": [],
+        # sentence_offsets     : [],
+        # token_offsets    : [],
+        "action": None,
+        "abstract": pubmed_xml["abstract"],
+        "authors": pubmed_xml["author_list"],
+        "keywords": [],
+        "introduction": None,
+        "discussion": None,
+        "references": []
+    }
+
+    make_jsonfile(case_report, json_path)
+
+def export_case_report_json(xml_path: str) -> json:
```

Comment:
Function name doesn't seem to make sense. It isn't really exporting anything. If anything its building a case_report dictionary object.